### DISCUSSION TITLE
环境变量中设置隐藏地区无效

### DISCUSSION THREAD

**Discussion by XiaoPika** [walinejs/waline#1233] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/10910848/181179817-f86f44c3-494c-434d-befe-ef14fc60ec0b.png)

![image](https://user-images.githubusercontent.com/10910848/181180269-34b94ae6-cbaf-426e-940f-29a528527533.png)

提前感谢百忙之中解惑的大佬，谢谢！

**Comment by lizheming** [comment, marked as answer]
退出登录后看下

**Reply by XiaoPika** [reply]
退出后仍然会显示地址，不过仅显示到省了。不知道能否彻底隐藏这个信息块呢？

![image](https://user-images.githubusercontent.com/10910848/181190077-5c0f321f-14e4-4a65-9978-3512d7dc1df2.png)

**Reply by lizheming** [reply]
那你看看有没有重启阿里云服务吧，现在看就是环境变量没有生效。阿里云我没怎么用过，你自己先研究下吧

**Reply by XiaoPika** [reply]
暂时通过 css 隐藏了，有空研究一下 Vercel 部署的设置为什么就是正常的。谢谢您！

**Reply by gumpBOBO** [reply]
> 那你看看有没有重启阿里云服务吧，现在看就是环境变量没有生效。阿里云我没怎么用过，你自己先研究下吧

请问下大大，我用的2.6.1 地域配置是不是默认隐藏的呢？ 需要自己+一下环境变量
